src/EditorFeatures/CSharpTest/CodeActions/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -2056,9 +2056,6 @@ static void Main(string[] args)
         System.Console.WriteLine([|^1|]);
     }
 }",
-@"
-
-using System;" +
```

Comment:
the reason for this is that the test literally adds the source code for System.Index at the top of hte file.  this code then triggers the conflict-detection logic because it's full of types from System, and we're about to add a `using System`.

Reply:
this is not a realistic user scenario.